$OpenBSD: patch-ext_pg_c,v 1.1 2010/12/20 17:31:08 jeremy Exp $

rb_yield_splat is not yet supported by rubinius, this is functionally
equivalent. Upstream fixed the issue in 9e11be78bfe4.

--- ext/pg.c.orig	Thu Dec  2 09:10:21 2010
+++ ext/pg.c	Thu Dec  2 09:12:32 2010
@@ -2172,7 +2172,7 @@ pgconn_wait_for_notify(int argc, VALUE *argv, VALUE se
 	PQfreemem( notification );
 
 	if ( rb_block_given_p() )
-		rb_yield_splat( rb_ary_new3(2, relname, be_pid) );
+		rb_yield_values(2, relname, be_pid);
 
 	return relname;
 }
